### PR TITLE
Adaptation methods

### PR DESCRIPTION
**Pull request by elzbietaZ**
- w baggingNaive muszę jeszcze przerobić tablicę votes na sparse matrix

### CODE REVIEW

File: skmultilearn/adapt/baggingNaive.py [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+        y = self.ensure_output_format(y, sparse_format='csc', enforce_sparse=True)
+
+        self.generate_partition(X, y)
+
+        self.classifiers = []
+
+        for i in xrange(self.model_count):
+            classifier = copy.deepcopy(self.classifier)
+            x_subset = self.generate_data_subset(X, self.partition[i], axis=0)
+            y_subset = self.generate_data_subset(y, self.partition[i], axis=0)
```
Comment (niedakh) [inline skmultilearn/adapt/baggingNaive.py]: maybe we could change the name self.partition to so self.subets and assume that in therms of row-wise decomposition we would use subsets and in label - wise decomposition - partition.

File: skmultilearn/adapt/baggingNaive.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+                for model in xrange(self.model_count):
+                    votes.append(predictions[model][instance][label])
+                assignLabel = self.vote(votes)
+                result[instance, label] = assignLabel
+
+        return result
+
+    def vote(self, votes):
+        assignLabel = 0
+        suma = sum(votes)
```
Comment (niedakh) [inline skmultilearn/adapt/baggingNaive.py]: english variable names please

File: skmultilearn/adapt/baggingNaive.py (hunk 1 above)
Comment (elzbietaZ) [inline skmultilearn/adapt/baggingNaive.py]: ok, good idea, that will be more clear

File: skmultilearn/adapt/baggingNaive.py (hunk 2 above)
Comment (elzbietaZ) [inline skmultilearn/adapt/baggingNaive.py]: Sorry, I forgot to change it

**Pull request merged**